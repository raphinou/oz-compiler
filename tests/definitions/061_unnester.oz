% Unnesting of variable assignment in declarations of a local and in proc argument
local
   ShowPlusOne
   A = 5-2+(8+4)*2
   B = A*2+4
in
   proc {ShowPlusOne Arg}
     {Show Arg+1}
   end
   {ShowPlusOne A}
   {ShowPlusOne B}
   {ShowPlusOne 2*A+3*B}
end
