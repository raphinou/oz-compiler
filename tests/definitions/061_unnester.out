28
59
229
